src/xdocs/checks/modifier/redundantmodifier.xml

```diff
@@ -153,6 +157,13 @@ public class ClassExtending extends ClassExample {
               <th>default value</th>
               <th>since</th>
             </tr>
+            <tr>
+              <td>jdkVersion</td>
+              <td>Set the JDK version that you are using.</td>
```

Comment:
Documentation does not mention how we support <= 8.

Reply:
done